services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> Movie:
+    queryset = Movie.objects.all()
+
+    if not genres_ids and not actors_ids:
+        return queryset
+
+    if genres_ids and actors_ids:
+        return queryset.filter(genres__in=genres_ids, actors__in=actors_ids)
+
+    if genres_ids:
+        return queryset.filter(genres__in=genres_ids)
+
+    if actors_ids:
+        return queryset.filter(actors__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
```

Comment:
The function `get_movie_by_id` is expected to return a single Movie instance, and the return type hint is correct in this case. However, it would be good to handle the case when there is no Movie with the provided id. In the current implementation, the `get()` method will raise a `Movie.DoesNotExist` exception. You may want to catch this exception and handle it appropriately, or let the user know that the movie was not found.